release-process.md

```diff
@@ -0,0 +1,32 @@
+# Cutting a new release of the Spin CRON trigger plugin
+
+To cut a new release of the CRON trigger plugin, you will need to do the following:
```

Comment:
nit: Is the all-caps CRON a sylistic flourish and nod to the TRON series?  If so, heartily approve.  Otherwise, perhaps just `Cron`?